### PR TITLE
config: create parent dir if missing

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**

### CODE REVIEW

**Comment by BusyJay** [comment]
CI still fails.

**Comment by zhangjinpeng1987** [comment]
Seems it is because can not connect to tidb-sever?

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -370,9 +370,22 @@ fn check_and_persist_critical_config(config: &TiKvConfig) {
         }
     }
 
+    // Create parent directory if missing.
+    if let Err(e) = fs::create_dir_all(&store_path) {
```
Comment (siddontang) [inline src/bin/tikv-server.rs]: do you add a test with no parent directory at first?

**Pull request merged**